src/collector.js

```diff
@@ -76,6 +77,10 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename) {
+    this.scannedFiles.push(filename);
```

Comment:
Categorising this list by scan type would definitely make it a lot more useful for debugging, yeah.